modules/cluster-logging-release-notes-5.4.0.adoc

```diff
@@ -0,0 +1,58 @@
+
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+[id="cluster-logging-ocp-compatibility"]
+= {product-title} compatibility
+The {logging-title} is provided as an installable component, with a distinct release cycle from the core {product-title}. The link:https://access.redhat.com/support/policy/updates/openshift#logging[Red Hat OpenShift Container Platform Life Cycle Policy] outlines release compatibility.
+
+[id="cluster-logging-release-notes-5-4-0"]
+= Version 5.4 of the {logging}
+The following advisories are available for version 5.4 of the {logging}:
+link:https://access.redhat.com/errata/RHSA-2022:1461[{logging-title-uc} Release 5.4]
+
+[id="openshift-logging-5-4-0-bug-fixes"]
+== Bug fixes
+
+*	Before this update, the `cluster-logging-operator` utilized cluster scoped roles and bindings to establish permissions for the Prometheus service account to scrape metrics. These permissions were only created when deploying the Operator using the console interface but were missing when deploying from the command line. This update fixes the issue by making the roles and bindings namespace-scoped. (link:https://issues.redhat.com/browse/LOG-2286[LOG-2286])
+
+*	Before this update, a prior change to fix dashboard reconciliation introduced a `ownerReferences` field to the resource across namespaces. As a result, both the config map and dashboard were not getting created in the namespace. With this update, the removal of the `ownerReferences` field resolves the issue and the OpenShift Logging dashboard is available in the console. (link:https://issues.redhat.com/browse/LOG-2163[LOG-2163])
```

Comment:
Maybe use a space character instead of a tab - I don't know if it matters.
```suggestion
* Before this update, a prior change to fix dashboard reconciliation introduced a `ownerReferences` field to the resource across namespaces. As a result, both the config map and dashboard were not getting created in the namespace. With this update, the removal of the `ownerReferences` field resolves the issue and the OpenShift Logging dashboard is available in the console. (link:https://issues.redhat.com/browse/LOG-2163[LOG-2163])
```

Reply:
Huh. That space isn't in the file itself. Just rendered. Removed & readded it just in case.